Dockerfile.jenkins

```diff
@@ -33,4 +33,4 @@ RUN npm install --production
 EXPOSE 8888
 
 # Launch application
-CMD [ "npm", "start", "--", "--port=8888", "--config=k8sdev", "--prefix", "/ui" ]
+# CMD [ "npm", "start", "--", "--port=8888", "--config=k8sdev", "--prefix", "/ui" ]
```

Comment:
Is this meant to be commented?

Reply:
Yes, the --config value must be dynamic and the rest should be as well. I'd rather control this through the vars in the deploy configs.

Reply:
Looking at this more and I may need switch to directly supporting ENV vars instead of passing these flags through the npm start command. Then we can set this to `CMD [ "npm", "start"]`

FYI @emuvente

Reply:
Then again...I did this hoping this would take care of starting the container:
https://github.com/kiva/ui/blob/master/deploy/charts/templates/deployment.yaml#L33

Is that a correct assumption? @JoeTravisKiva @leoperry

Reply:
So I think that's a correct assumption, but I believe that it would be better to apply the environment specific config through ENV variables as you mention above. That's how we do it in the graphql federation service, and it also gives more flexibility in being able to override the values at runtime.

Also, not entirely on the same subject, but we also use https://github.com/krallin/tini as an entry point, which we got from https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md

Reply:
@JoeTravisKiva I'll look into adding Tini at a later time

I'm going to try to start the app using the templated npm start script for now as changing to ENV variables will require refactoring portions of the Ui app.